app/backend/src/couchers/models.py

```diff
@@ -618,11 +619,14 @@ class GroupChatSubscription(Base):
     def unseen_message_count(self):
         return (
             Session.object_session(self)
-            .query(Message.id)
-            .join(GroupChatSubscription, GroupChatSubscription.group_chat_id == Message.conversation_id)
-            .filter(GroupChatSubscription.id == self.id)
-            .filter(Message.id > GroupChatSubscription.last_seen_message_id)
-            .count()
+            .execute(
+                select(func.count())
+                .select_from(Message)
+                .join(GroupChatSubscription, GroupChatSubscription.group_chat_id == Message.conversation_id)
+                .filter(GroupChatSubscription.id == self.id)
+                .filter(Message.id > GroupChatSubscription.last_seen_message_id)
+            )
+            .scalar_one()
         )
```

Comment:
Move this function out of models.py to eliminate the circular import problem (i.e. do no select in models.py)

Reply:
LMK what you think of the new method

Reply:
hmm, was this property unused? Even easier :+1: 

I like it a lot that models.py does not deal with any session now.

Reply:
It was used, but only in conversations, which is where I put it.

Reply:
>I like it a lot that models.py does not deal with any session now.

Ya, that was a cool added benefit